src/tracing/trace_event.h

```diff
@@ -244,9 +244,20 @@ enum CategoryGroupEnabledFlags {
 
 // Adds a trace event with a given id, thread_id, and timestamp. Not
 // Implemented.
-#define INTERNAL_TRACE_EVENT_ADD_WITH_ID_TID_AND_TIMESTAMP(            \
-    phase, category_group, name, id, thread_id, timestamp, flags, ...) \
-  UNIMPLEMENTED()
+#define INTERNAL_TRACE_EVENT_ADD_WITH_ID_TID_AND_TIMESTAMP(                  \
```

Comment:
Is a sync with upstream?

Reply:
In sync with upstream? Nope, not yet. But that reminds me... 

@ofrobots ... ^^ To implement this I needed this macro implemented also.

Reply:
@jasnell created https://chromium-review.googlesource.com/c/v8/v8/+/924507 that adds the same macro upstream.

Note that for the trace-event macros, a sync with upstream is no longer a necessity. Our copy of `trace_event.h` is derived from upstream, but has a slightly different implementation. It is okay for them to pick up features at a different pace. Please do continue to ping if we add more macros locally. There is no need to block on upstream.